06_gpu_and_ml/stable_diffusion_cli.py

```diff
@@ -79,14 +79,20 @@ def download_models():
 
 
 image = (
-    modal.Image.conda()
-    .run_commands(
+    modal.Image.debian_slim()
+    .pip_install(
         [
-            "conda install xformers -c xformers/label/dev",
-            "conda install pytorch torchvision pytorch-cuda=11.7 -c pytorch -c nvidia",
+            "accelerate",
+            "diffusers[torch]>=0.10",
+            "ftfy",
+            "torch",
+            "torchvision",
+            "transformers",
+            "triton",
+            "safetensors",
+            "xformers==0.0.16rc393",
```

Comment:
we have to use a release candidate here – the last non-RC one is 0.0.13, and it doesn't work